### PR TITLE
Fix acceptance tests failures caused by fetch 50 and a race in the tests

### PR DESCRIPTION
**Pull request by mertyildiran**
Fixes the acceptance tests that became unstable after https://github.com/up9inc/mizu/pull/1056

Fixes `{alt+enter}` to `{ctrl+enter}` in the acceptance tests.

### CODE REVIEW

File: acceptanceTests/cypress/integration/tests/UiTest.js [hunk 1]
```diff
@@ -182,39 +182,53 @@ function checkIllegalFilter(illegalFilterName) {
...
     } = filterDetails;
 
     const entriesForDeeperCheck = 5;
 
-    it(`checking the filter: ${name}`, function () {
+    it(`checking the filter: ${filter}`, function () {
+        // wait half a second and pause the stream to preserve the DOM
+        cy.wait(500);
+        cy.get('#pause-icon').click();
+        cy.waitUntil(function() {
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/UiTest.js:199]: why is this needed? 
cy.get('#pause-icon').should('not.be.visible') waits for the pause icon to not be visible

File: acceptanceTests/cypress/integration/tests/UiTest.js [hunk 2]
```diff
@@ -182,39 +182,53 @@ function checkIllegalFilter(illegalFilterName) {
...
-        applyByEnter
+        applyByCtrlEnter
     } = filterDetails;
 
     const entriesForDeeperCheck = 5;
 
-    it(`checking the filter: ${name}`, function () {
+    it(`checking the filter: ${filter}`, function () {
+        // wait half a second and pause the stream to preserve the DOM
+        cy.wait(500);
```
[the remark this answers is not in the document]
Comment (mertyildiran) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @RoyUP9 waits for fetch 50 (added with https://github.com/up9inc/mizu/pull/1056) to settle and solves [this error](https://github.com/up9inc/mizu/runs/6356341461?check_suite_focus=true#step:9:823).

File: acceptanceTests/cypress/integration/tests/UiTest.js (hunk 1 above)
Comment (mertyildiran) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @RoyUP9 I didn't want to add more recursive promises and further complicate the code. So, I used https://www.npmjs.com/package/cypress-wait-until instead;

> **Why did we write it?**
A lot of StackOverflow users had some difficulties in implementing a recursive promise with Cypress just to repeatedly check for something to happen (see two of the various questions about the topic: How can i wait for each element in a list to update to a certain text? And How do I wait until a cookie is set?).
This plugin is dedicated to them ❤️

File: acceptanceTests/cypress/integration/tests/UiTest.js [hunk 3]
```diff
@@ -182,39 +182,53 @@ function checkIllegalFilter(illegalFilterName) {
...
                 // applying the filter with alt+enter or with the button
-                cy.get('.w-tc-editor-text').type(`${name}${applyByEnter ? '{alt+enter}' : ''}`);
+                cy.get('.w-tc-editor-text').type(`${filter}${applyByCtrlEnter ? '{ctrl+enter}' : ''}`);
                 cy.get('.w-tc-editor').should('have.attr', 'style').and('include', Cypress.env('greenFilterColor'));
-                if (!applyByEnter)
+                if (!applyByCtrlEnter)
                     cy.get('[type="submit"]').click();
 
+                // wait half a second and pause the stream to preserve the DOM
+                cy.wait(500);
```
[the remark this answers is not in the document]
Comment (mertyildiran) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @RoyUP9 done with https://github.com/up9inc/mizu/commit/314a7c60ca33610c5352d70517b649ef4eb4baf9

File: acceptanceTests/cypress/integration/tests/UiTest.js (hunk 1 above)
Comment (RoyUP9) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: Isn't cypress default behavior to wait X timeout for the should part to be true (without the waitUntill)?

Comment (mertyildiran) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @RoyUP9 `cypress-wait-until` dependency is removed with https://github.com/up9inc/mizu/commit/4f0a67feca968103f1a00afd2bed5a4508129862

File: acceptanceTests/cypress/integration/tests/UiTest.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (lirazyehezkel) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @mertyildiran I saw we have a timeout of 3000 ms for this operation of opening the WS and fetching the first 50 entries, how can make sure the 500ms waiting is enough?

[the remark this answers is not in the document]
Comment (mertyildiran) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @lirazyehezkel the queries in the test cases settle under 500ms in practice. For 3000ms there has to be 50k+ records and a strict filter that barely matches any recent records.

[the remark this answers is not in the document]
Comment (lirazyehezkel) [reply acceptanceTests/cypress/integration/tests/UiTest.js]: @mertyildiran got it, thx
Approved from my side, @RoyUP9 you can approve if you don't have any other comments

**Review by RoyUP9 — APPROVED** [review]
